packages/pwa-kit-dev/src/ssr/server/build-dev-server.js

```diff
@@ -278,11 +278,11 @@ export const DevServerMixin = {
 
     render(req, res, next) {
         const app = req.app
-        if (app.__webpackReady()) {
+
+        // Ensure that we do not try to render anything until the webpack bundle is valid
```

Comment:
nitpicking, the comment explains the "what".  It would be nice to document the "why". Why do we not want to render anything? what's the harm to end users?

Reply:
it might be very obvious to you as someone who is an expert, but comments are mostly useful for developers new to the code base and the "why"s are particularly useful to them!